src/current/molt/migrate-data-load-replicate-only.md

```diff
@@ -0,0 +1,159 @@
+	--mode data-load
+	~~~
+	</section>
+
+	<section class="filter-content" markdown="1" data-scope="oracle">
+	The command assumes an Oracle Multitenant (CDB/PDB) source. `--source-cdb` specifies the container database (CDB) connection string.
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \
+	--source-cdb $SOURCE_CDB \
+	--target $TARGET \
+	--schema-filter 'migration_schema' \
+	--table-filter 'employees|payments|orders' \
+	--bucket-path 's3://migration/data/cockroach' \
+	--table-handling truncate-if-exists \
+	--mode data-load
+	~~~
+	</section>
+
+{% include molt/fetch-data-load-output.md %}
+
+## Verify the data load
+
+Use [MOLT Verify]({% link molt/molt-verify.md %}) to confirm that the source and target data is consistent. This ensures that the data load was successful.
+
+{% include molt/verify-output.md %}
+
+## Replicate changes to CockroachDB
+
+With initial load complete, start replication of ongoing changes on the source to CockroachDB.
+
+<a id="replication-flags"></a>
+{% include molt/fetch-replicator-flags.md %}
+
+1. Issue the [MOLT Fetch]({% link molt/molt-fetch.md %}) command to start replication on CockroachDB, specifying [`--mode replication-only`]({% link molt/molt-fetch.md %}#fetch-mode). For details on `--replicator-flags`, refer to [Replication flags](#replication-flags). In this example, the `--metricsAddr :30005` replication flag enables a Prometheus endpoint at `http://localhost:30005/_/varz` where replication metrics will be served. You can use these metrics to [verify that replication has drained](#stop-replication-and-verify-data) in a later step.
+
+	<section class="filter-content" markdown="1" data-scope="postgres">
+	Be sure to specify the same `--pglogical-replication-slot-name` value that you provided in [Load data into CockroachDB](#load-data-into-cockroachdb).
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \ 
+	--target $TARGET \
+	--table-filter 'employees|payments|orders' \
+	--pglogical-replication-slot-name cdc_slot \
+	--replicator-flags '--metricsAddr :30005' \
+	--mode replication-only
+	~~~
+	</section>
+
+	<section class="filter-content" markdown="1" data-scope="mysql">
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \ 
+	--target $TARGET \
+	--table-filter 'employees|payments|orders' \
+	--non-interactive \
+	--replicator-flags '--defaultGTIDSet 4c658ae6-e8ad-11ef-8449-0242ac140006:1-29 --metricsAddr :30005' \
+	--mode replication-only
+	~~~
+	</section>
+
+	<section class="filter-content" markdown="1" data-scope="oracle">
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source $SOURCE \
+	--source-cdb $SOURCE_CDB \
+	--target $TARGET \
+	--schema-filter 'migration_schema' \
+	--table-filter 'employees|payments|orders' \
+	--replicator-flags '--backfillFromSCN 26685444 --scn 26685786 --metricsAddr :30005 --userscript table_filter.ts' \
```

Comment:
I see we have the userscript inner flag here but not for the other examples, should we also add this `--userscript` in the mysql and pg examples?

Reply:
Added to MySQL. According to @Jeremyyang920 this isn't needed for PG.